tests/network-tests/src/scenarios/full.ts

```diff
@@ -126,4 +127,5 @@ scenario('Full', async ({ job, env }) => {
   job('init storage and distribution buckets via CLI', [initDistributionBucket, initStorageBucket]).after(
     contentDirectoryJob
   )
+  job('updating member verify account', updateValidatorAccount).after(hireLeads)
```

Comment:
This is correct but for organization sake please move this just under the `updating member verification status` job